### PR TITLE
feat(shipyard-controller): Documentation & Feature Flag

### PR DESCRIPTION
**Pull request by asamonik**
## This PR

- adds documentation to the DebugUI
- adds a feature flag to the DebugUI to enable it

### CODE REVIEW

File: shipyard-controller/main.go [hunk 1]
```diff
@@ -294,13 +294,19 @@ func _main(env config.EnvConfig, kubeAPI kubernetes.Interface) {
 		Handler: engine,
 	}
 
-	// TODO! feature flag
-	/*
+	debuguiEnabled := os.Getenv("DEBUGUI_ENABLED")
```
Comment (bacherfl) [inline shipyard-controller/main.go:297]: this should be retrieved from the envConfig object

File: shipyard-controller/debugdocs/docs.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+The Debug-UI was built to debug keptn.
+
+**Features:**
+ - See all sequences of a project 
+ - Get all the events of a specific sequence 
+ - Get the sequences which are blocking a specific sequence 
+ - Export a MongoDB collection
+
+The Debug-UI can be exposed on port 9090 of the shipyard controller by setting 'DEBUGUI_ENABLED' in the deploy/service.yml to "true".
```
Comment (bacherfl) [inline shipyard-controller/debugdocs/docs.md:11]: instead of referring specifically to the `deploy/service.yaml` I would suggest to rephrase it to something like "set the `DEBUGUI_ENABLED environment variable to `"true"`"

**Comment by odubajDT** [comment]
closing in favor of https://github.com/keptn/keptn/pull/8736
